apps/kg-explorer/src/app/components/provenance-menu/provenance-menu.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { provideHttpClient } from '@angular/common/http';
+import { render } from '@testing-library/angular';
+import { saveAs } from 'file-saver';
+
+import { ProvenanceMenuComponent } from './provenance-menu.component';
+
+jest.mock('file-saver');
+
+describe('ProvenanceMenuComponent', () => {
+  it('should render', async () => {
+    const promise = render(ProvenanceMenuComponent, {
```

Comment:
Create a small `setup` function that does the common logic instead of repeating the same logic in each test